### DISCUSSION TITLE
An unofficial client for excalidraw

### DISCUSSION THREAD

**Discussion by m1911star** [excalidraw/excalidraw#5838] [Show and tell]
Hi there，I just build a client based on tauri. Feel free to clone at https://github.com/m1911star/excalidraw-client

Excalidraw is a wonderful tool for me and thank you all~

![image](https://user-images.githubusercontent.com/4948120/200173460-74f1d240-dadd-4f34-9570-ee20589304d7.png)

This is my local build version, still working on release version, will publish soon~
[Excalidraw_0.1.0_x64_en-US.zip](https://github.com/excalidraw/excalidraw/files/9946082/Excalidraw_0.1.0_x64_en-US.zip)

**Comment by m1911star** [comment]
https://github.com/m1911star/excalidraw-client/releases/tag/excalidraw-client-v0.1.0-alpha

**Reply by weyderfs** [reply]
@m1911star the version is outdated, there's a possibility of update?
